subject_files_status_for_dpdash.py

```diff
@@ -34,6 +35,27 @@ def _latest_mtime(p: Path) -> str:
     return datetime.fromtimestamp(latest).strftime('%Y-%m-%d')
 
 
+def check_upenn_cnb(subject_raw_path: Path) -> bool:
+    '''Check if there are SPLLT-A and NOSPLLT in the Penn CNB json file'''
+    json_paths = list((subject_raw_path / 'surveys').glob('*UPENN.json'))
+
+    if len(json_paths) == 0:  # No file
+        return False
```

Comment:
Here and below you return a `bool` but our previous `_is_file()` returned `1` or `0`:

https://github.com/AMP-SCZ/utility/blob/c55918b58bcf68a18a1ba024ccead04f62bd6bbb/subject_files_status_for_dpdash.py#L201-L203

Should you revise?

Reply:
updated to return `int`